tests/rpc/personal_test.go

```diff
@@ -148,9 +148,13 @@ func TestPersonal_LockAccount(t *testing.T) {
 func TestPersonal_Unpair(t *testing.T) {
 	t.Skip("skipping TestPersonal_Unpair")
 
-	rpcRes := Call(t, "personal_unpair", []interface{}{"", 0})
+	_, err := CallWithError(t, "personal_unpair", []interface{}{"", 0})
+	require.True(t, errors.Is(err, fmt.Errorf("smartcard wallet not supported yet")))
+}
 
-	var res error
-	err := json.Unmarshal(rpcRes.Result, &res)
+func TestPersonal_InitializeWallet(t *testing.T) {
+	t.Skip("skipping TestPersonal_InitializeWallet")
```

Comment:
same

Reply:
I tried the testcases in personal_test.go with all not skipped for live ethermintd.
every testcase failed for method not registered.
And for testcase like TestPersonal_LockAccount, it expects an error when trying to sign a locked account.
So I just follow the pattern.

Reply:
add entry in changelog and remove skip